### ISSUE TITLE
NFS Undercover won't fullscreen if Nine is used.

### ISSUE THREAD

**Issue by Ufeldraku** [iXit/Mesa-3D#226]
As title says, NFSU won't fullscreen with Nine, but it's fine with staging though.

**Comment by axeldavy** [comment]
Weird, I have already played the game fullscreen on nine.

**Comment by siro20** [comment]
What happens if you want to run it in fullscreen mode ?
Does it output an error message ?

**Comment by Ufeldraku** [comment]
Nothing, the output is the same, with staging it fullscreens properly
With nine, In gnome and Mate, it just changes the resolution of my screen, but NFSU's window won't fullscreen, the topbar is still present.
Using that option that disables the window manager  ability to manage wine's windows, it makes the window fullscreen properly, but it doesn't let me give any input on the screen, I can click, but the keyboard won't work.
And yeah, In previous versions it used to fullscreen properly, but it would crash, now it won't fullscreen properly, but it won't crash.

**Comment by cRaZy-bisCuiT** [comment]
Affects me as well. Exactly the same as mentioned above.

**Comment by axeldavy** [comment]
If I understand cRaZy-bisCuiT is using arch wine-gaming. I had a few games not launch properly with it, but work properly with my wine + nine only builds. What package are you using Ufeldraku ?

**Comment by cRaZy-bisCuiT** [comment]
Yes, you're right. I'm using the AUR version ... isn't that one grabbed from git? I'll reinstall the game once more because problems (also with native windows) had some trouble on some hardware every once in a while.

**Comment by xhebox** [comment]
AUR version is actually based on releases from 'sarnex/wine-d3d9-patches' and 'wine-compholio/wine-patched' on github.

**Comment by cRaZy-bisCuiT** [comment]
[code]
▶ pacman -Q | grep wine
wine-gaming-nine 1.9.22-1
winetricks 20161012-1
[/code]

Is your version up at the AUR as well?

**Comment by sarnex** [comment]
I updated my patches, please try rebuilding wine.

**Comment by Ufeldraku** [comment]
Um, axeldavy, yeah, i used to use the wine-gaming-nine.
